ui/tests/acceptance/open-api-path-help-test.js

```diff
@@ -116,9 +115,8 @@ function authEngineHelper(test, authBackend) {
         const modelName = `generated-${itemName}-${authBackend}`;
         // Generated items need to instantiate the model first via getNewModel
         await this.pathHelp.getNewModel(modelName, this.mount, `auth/${this.mount}/`, itemName);
-        const model = this.store.createRecord(modelName, {});
-        // Generated items don't have this method -- helpUrl is calculated in path-help.js line 101
-        const helpUrl = model.getHelpUrl(this.mount);
+        // Generated items don't have helpUrl method -- helpUrl is calculated in path-help.js line 101
```

Comment:
thank you for this comment! do we have test coverage that assures removing the `|| newModel.proto().getHelpUrl(backend);` of the conditional in path-help is alright? 

From what I gather it seems like all non-generated items have paths defined in the new helper, `getHelpUrlForModel` and any generated items should otherwise follow the pattern of `/v1/${apiPath}${path.slice(1)}?help=true` - just wanted to double check!

Reply:
Good question! I removed the `||` because the first statement would never be falsey. 
```
// original
helpUrl = `/v1/${apiPath}${path.slice(1)}?help=true` || newModel.proto().getHelpUrl(backend);
```
even if `apiPath` and `path.slice(1)` resolve to empty strings, the first part will resolve at minimum to `/v1/?help=true`. 

Otherwise your comment is true -- if we have the base model for it, the helpUrl should be defined in `getHelpUrlForModel`. If it's generated (like for userpass users) then it follows the pattern in line 101 of path-help service

Reply:
Also, this is a skipped test -- I just didn't want any instances of `getHelpUrl` laying around so I could be sure I got them all 😂

Reply:
great thinking! nice to keep up with tech debt while we have context. awesome! that all makes sense, thanks for explaining.